app.js

```diff
@@ -0,0 +1,192 @@
+const playerController = (() => {
+  const Player = function (name, symbol) {
+    this.name = name;
+    this.moves = [];
+    this.symbol = symbol;
+    this.winner = false;
+    this.active = false;
+  };
+  return {
+    addPlayer: function (name, symbol) {
+      return new Player(name, symbol);
+    },
+    addMove: function (move) {
+      this.moves.push(move);
+    },
+
+    addWinener: function () {
+      this.winner = true;
+    },
+
+    activePlayer: function (active) {
+      this.active = active;
+    },
+  };
+})();
+
+const gameBoardController = (function () {
+  const GameBoard = function (xPlayer, oPlayer) {
+    this.xPlayer = xPlayer;
+    this.oPlayer = oPlayer;
+    this.status = "Running";
+    this.currentPlay = xPlayer;
+    this.winner = null;
+    this.playCount = 0;
+  };
+
+  const boardWinner = function (game) {
+    const board = document.getElementsByClassName("box");
+    document.getElementById("game-status").innerHTML = "status = running";
+
+    for (let i = 0; i < board.length; i++) {
+      if (game.playCount < 9) {
+        board[i].addEventListener("click", function () {
+          if (board[i].innerHTML.trim() === "" && game.status === "Running") {
+            board[i].innerHTML = game.currentPlay.symbol;
+            game.currentPlay =
+              game.currentPlay.symbol === "X" ? game.oPlayer : game.xPlayer;
+            document.getElementById("player").innerHTML = game.currentPlay.name;
+
+            if (
+              board[0].innerHTML === board[1].innerHTML &&
+              board[1].innerHTML === board[2].innerHTML &&
+              board[0].innerHTML.trim() !== ""
+            ) {
+              showWinner(game);
+            } else if (
+              board[3].innerHTML === board[4].innerHTML &&
+              board[4].innerHTML === board[5].innerHTML &&
+              board[3].innerHTML.trim() !== ""
+            ) {
+              showWinner(game);
+            } else if (
+              board[6].innerHTML === board[7].innerHTML &&
+              board[7].innerHTML === board[8].innerHTML &&
+              board[6].innerHTML.trim() !== ""
+            ) {
+              showWinner(game);
+            } else if (
+              board[0].innerHTML === board[3].innerHTML &&
+              board[3].innerHTML === board[6].innerHTML &&
+              board[0].innerHTML.trim() !== ""
+            ) {
```

Comment:
I think it is important not to miss up the app logic with DOM.

We can create an array (or object) in the `gameBoard` function that stores the plays from each player and a method that keeps tracks of this array to know when there's a winner or a draw.

This information should be enough to be used to update the state of the app whenever a player makes a **valid move**, we can also add feedback for **invalid moves**.


> * [ ] It has a function with the only responsibility of rendering the contents of the gameboard array to the webpage ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/tic_tac_toe/render_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/tic-tac-toe-javascript))
> * [ ] It has a function that that allows a player to 'mark' a spot on the board ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/tic_tac_toe/mark_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/tic-tac-toe-javascript))
> * [ ] It has a method(s) to check the game's winning conditions ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/tic_tac_toe/win_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/tic-tac-toe-javascript))


This was also discussed by previous reviewers in https://github.com/gitnyasha/tic-tac-toe-javascript/pull/1#discussion_r428587246